modules/http/src/main/java/org/mule/module/http/internal/listener/HttpRequestToMuleEvent.java

```diff
@@ -122,6 +122,15 @@ else if (entity instanceof InputStreamHttpEntity)
         }
 
         final DefaultMuleMessage defaultMuleMessage = new DefaultMuleMessage(payload, inboundProperties, outboundProperties, inboundAttachments, muleContext);
+
+        for (String property : inboundProperties.keySet())
```

Comment:
Yes, refactoring the code to make use of that line is out of scope (there are 4 levels of "if" in that code, I prefer to keep them untouched)